src/styles/_mixins.scss

```diff
@@ -19,3 +19,31 @@
     }
   }
 }
+
+@mixin generateCardClasses() {
+  @each $cardType, $cardColor in $cardTypes {
+    .card--#{$cardType}.card {
+      border-color: map-get($colors, '#{$cardColor}800');
+    }
+    @each $tagType in $tagTypes {
+      @if $tagType == default {
+        .card--#{$cardType}-#{$tagType}.card__tag {
+          color: map-get($colors, '#{$cardColor}100');
+          background-color: map-get($colors, '#{$cardColor}800');
+        }
+      } @else if $tagType == minor {
+        .card--#{$cardType}-#{$tagType}.card__tag {
+          color: map-get($colors, '#{$cardColor}800');
+          background-color: map-get($colors, '#{$cardColor}100');
+        }
+      } @else {
+        .card--#{$cardType}-#{$tagType}.card__tag {
+          color: map-get($colors, '#{$cardColor}800');
+          border: 1px solid map-get($colors, '#{$cardColor}800');
+          display: grid;
+          align-content: center;
+        }
+      }
+    }
+  }
+}
```

Comment:
To zdecydowanie powinno być w pliku z `.card` jako, że to są jego modyfikatory. Nie ma żadnej zalety wyciągania tego do `@mixin`. Jako, że używasz tutaj _explicit_ `.card` to taki `@mixin` i tak nie jest reużywalny.